### PR TITLE
MODINREACH-227 - Create API endpoint (GET) for location mappings

### PR DESCRIPTION
**Pull request by s1mpleGit**
[MODINREACH-227](https://issues.folio.org/browse/MODINREACH-227) - Create API endpoint (GET) for location mappings
## Purpose
Create an endpoint to get location-mappings for all libraries in a single request.

## Approach
Endpoint implemented

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

### CODE REVIEW

File: src/main/java/org/folio/innreach/domain/service/LocationMappingService.java [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
   LocationMappingsDTO getAllMappings(UUID centralServerId, UUID libraryId, int offset, int limit);
 
+  LocationMappingsDTO getAllMappingsForAllLibraries(UUID centralServerId, int offset, int limit);
+
```
Comment (oliinyko) [inline src/main/java/org/folio/innreach/domain/service/LocationMappingService.java]: rename `getAllMappings` -> `getMappingsByLibraryId`
rename `getAllMappingsForAllLibraries` -> `getAllMappings`

File: src/test/java/org/folio/innreach/controller/LocationMappingControllerTest.java [hunk 2]
```diff
@@ -353,10 +378,18 @@ private static String baseMappingURL() {
     return baseMappingURL(PRE_POPULATED_CENTRAL_SERVER_ID, PRE_POPULATED_LIBRARY_ID);
   }
 
+  private static String baseMappingURLForAllLibraries() {
+    return baseMappingURLForAllLibraries(PRE_POPULATED_CENTRAL_SERVER_ID);
+  }
```
Comment (oliinyko) [inline src/test/java/org/folio/innreach/controller/LocationMappingControllerTest.java]: just put the logic of `baseMappingURLForAllLibraries(String serverId)` in this method

File: src/main/java/org/folio/innreach/domain/service/LocationMappingService.java (hunk 1 above)
Comment (s1mpleGit) [reply src/main/java/org/folio/innreach/domain/service/LocationMappingService.java]: refactored

File: src/test/java/org/folio/innreach/controller/LocationMappingControllerTest.java (hunk 2 above)
Comment (s1mpleGit) [reply src/test/java/org/folio/innreach/controller/LocationMappingControllerTest.java]: refactored

**Comment by oliinyko** [comment]
add the endpoint to module descriptor

**Comment by s1mpleGit** [comment]
> add the endpoint to module descriptor

thanks

**Review by oliinyko — APPROVED** [review]

**Pull request merged**